src/utils/update/beneficial_owners_managing_officers_data_fetch.ts

```diff
@@ -76,3 +75,39 @@ export const retrieveManagingOfficers = async (req: Request, appData: Applicatio
     }
   }
 };
+
+export const getBoPrivateData = async (req: Request, appData: ApplicationData): Promise<BeneficialOwnersPrivateData> => {
+  const transactionId = appData.transaction_id;
+  const overseasEntityId = appData.overseas_entity_id;
+  let boPrivateData: BeneficialOwnersPrivateData = {
+    boPrivateData: []
+  };
+
+  if (transactionId && overseasEntityId){
+    try {
+      boPrivateData = await getBeneficialOwnerPrivateData(req, transactionId, overseasEntityId) as BeneficialOwnersPrivateData;
+    } catch (error) {
+      logger.errorRequest(req, "Private Beneficial Owner details could not be retrieved for overseas entity " + appData.entity_number);
+    }
+  }
+  return boPrivateData;
+};
+
+export const mapBeneficialOwnerIndvidual = (psc: CompanyPersonWithSignificantControl, boPrivateData: BeneficialOwnersPrivateData, appData: ApplicationData) => {
+  const individualBeneficialOwner = mapPscToBeneficialOwnerTypeIndividual(psc);
+  individualBeneficialOwner.usual_residential_address = mapBoPrivateAddress(boPrivateData, individualBeneficialOwner.ch_reference!);
```

Comment:
The ch_reference could be undefined so we should either check this exists either before calling `mapBoPrivateAddress` or inside the function (and pass the individualBeneficialOwner data)

Reply:
we have the null assertion , but if you would prefer an if statement to avoid the lint warning we could do that!